### ISSUE TITLE
Island model

### ISSUE THREAD

**Issue by txt2vz** [jenetics/jenetics#782]
Would be nice if there was an easy way to implement an island model such that evolving sub-populations (possibly with different parameters) could exchange a small number of individuals after a set number of generations.

**Comment by jenetics** [comment]
Yes, that would be nice ;-) I already explored the _island model_ myself. Since Jenetics has all building blocks to do this, it is not that easy to implement a _general_ usable solution.
If you are interested, the main build block would be the [Engine.stream​(Iterable<Genotype<G>> genotypes)](https://www.javadoc.io/static/io.jenetics/jenetics/6.3.0/io/jenetics/engine/EvolutionStreamable.html#stream(java.lang.Iterable)) method, which allows to create a new [EvolutionStream](https://www.javadoc.io/doc/io.jenetics/jenetics/latest/io/jenetics/engine/EvolutionStream.html) from a given initial population. Have you already tried to implement an _island model_? Would be nice if you can share your ideas/implementations.
